crates/net/src/http/request.rs

```diff
@@ -38,10 +38,9 @@ impl RequestBuilder {
     }
 
     /// Set the body for this request.
-    pub fn body(mut self, body: impl Into<JsValue>) -> Result<Request, Error> {
+    pub fn body(mut self, body: impl Into<JsValue>) -> Self {
```

Comment:
I didn't see a reason why Request `body` should `build` , this allows us to set the body in any order.

I will add a test to verify this is sound.